pkgs/applications/networking/sniffers/wireshark/default.nix

```diff
@@ -47,13 +54,23 @@ stdenv.mkDerivation {
     categories = "Network;System";
   };
 
+  libPath = stdenv.lib.makeLibraryPath ([
+    stdenv.cc.cc stdenv.cc.libc
+    libpcap glib libnl zlib libcap
+  ] ++ optional withQt qt4
+    ++ optional withGtk gtk);
+
   postInstall = optionalString (withQt || withGtk) ''
     mkdir -p "$out"/share/applications/
     mkdir -p "$out"/share/icons/
     cp "$desktopItem/share/applications/"* "$out/share/applications/"
     cp image/wsicon.svg "$out"/share/icons/wireshark.svg
-  '' + optionalString withQt ''
-    mv "$out/bin/wireshark-qt" "$out/bin/wireshark"
+  '';
+
+  postFixup = ''
```

Comment:
Ditto, these settings should be superfluous if the prefix is set properly at configure time.